spec/controllers/conferences_controller_spec.rb

```diff
@@ -24,10 +24,31 @@
   end
 
   describe 'new' do
-    it 'redirects unauthorized users' do
-      get :new
-      expect(response).to redirect_to orga_conferences_path
-      expect(Conference).to_not receive(:create)
+    context 'unauthenticated users' do
+      it 'cannot create new conference' do
+        get :new
+        expect(response).to redirect_to(root_url)
+      end
+    end
+
+    context 'student logged in' do
+      let!(:student) { FactoryGirl.create(:student) }
+      let(:team) { student.student_team }
+      before :each do
+        sign_in student
+      end
+
+      it 'are allowed to visit a new conference view' do
+        get :new
+        expect(response).to render_template("new")
+      end
+
+      it 'are allowed to create a new conference' do
+        expect{
+          post :create, params: { team_id: team.id ,conference: {name: 'name', country: 'country', region: 'region', location: 'location', city: 'city', season_id:'1'}}
```

Comment:
For better readability, I recommend putting the conference params into a let variable